vmdb/app/models/orchestration_template.rb

```diff
@@ -60,6 +60,20 @@ def eligible_managers
     self.class.eligible_managers
   end
 
+  # return the validation error message; otherwise nil
+  def validate_content(manager = nil)
+    test_managers = manager.nil? ? eligible_managers : [manager]
+    test_managers.each do |mgr|
+      begin
+        return mgr.orchestration_template_validate(self)
+      rescue
```

Comment:
If you are going to do this you can use an inline rescue: `return mgr.orchestration_template_validate(self) rescue nil`